### DISCUSSION TITLE
Welcome to the Keygen community!

### DISCUSSION THREAD

**Discussion by ezekg** [keygen-sh/community#1] [General]
## 👋 Welcome!

Here at [Keygen](https://keygen.sh/), we’re going to be giving GitHub's new Discussions a spin. We'll be using it as a place to connect with members of our community and to help out where we can. We hope that you're able to:

* Ask questions about integrating Keygen or about licensing in general.
* Ask for help about an issue you're hitting.
* Share ideas or feature requests.
* Engage with other community members.

Feel free to introduce yourself here, or go ahead and create a new discussion.

[*keygen noises intensify*](https://www.youtube.com/watch?v=FVX6t-ivMfE)

**Comment by zakmuh** [comment]
Hi All,

I'm Zak, I'm from UK and I'm just a newbie here. Just popped in to say Hi and introduce myself.

Have fun guys.

Cheers
